### PR TITLE
Add is_pred_labels to struct labeler and increment version

### PR DESCRIPTION
**Pull request by JGSweets**
Allows user to specify is_pred_labels (False, True) in post processor for structured.
Now labeler can return indexes instead of labels

### CODE REVIEW

File: dataprofiler/labelers/data_processing.py [hunk 1]
```diff
@@ -1231,6 +1232,9 @@ def __init__(self, default_label='UNKNOWN', pad_label='PAD',
         :param flatten_separator: separator used to put between flattened
             samples.
         :type flatten_separator: str
+        :param is_pred_labels: (default: true) if true, will convert the model
+            indexes to the label strings given the label_mapping
```
Comment (ksneab7) [inline dataprofiler/labelers/data_processing.py:1236]: nit-pick, but indices not indexes.

**Review by ksneab7 — APPROVED** [review]
Very small comment about a function headers english but looks good to me

File: dataprofiler/labelers/data_processing.py (hunk 1 above)
Comment (JGSweets) [reply dataprofiler/labelers/data_processing.py:1236]: https://stackoverflow.com/questions/1378781/proper-terminology-should-i-say-indexes-or-indices

**Review by taylorfturner — APPROVED** [review]
LGTM

**Pull request merged**